domain-http/interface/src/main/java/org/jboss/as/domain/http/server/ManagementHttpServer.java

```diff
@@ -113,6 +118,7 @@
     private volatile AcceptingChannel<SslConnection> secureServer;
     private final SSLContext sslContext;
     private final SslClientAuthMode sslClientAuthMode;
+    private static Map<Pattern, Charset> userAgentCharsetMap = generateCharsetMap();
```

Comment:
This needs the 'final' keyword.

Also, if you don't mind, I'd prefer that it's declared above the instance fields. That's a minor codestyle nit.